gtests/net/mptcp/dss/dss_ssn_specified_client.pkt

```diff
@@ -24,4 +24,5 @@
 +0.0   < .  1001:1001(0) ack 101 win 450 <dss dack8=101 nocs>
 +0.4 close(3) = 0
 +0.0   > . 101:101(0) ack 1001 <nop, nop,TS val 100 ecr 700,dss dack8=1001 dsn8=101 ssn=0 dll=1 nocs fin, nop, nop>
-+0.0   > F. 101:101(0) ack 1001 <nop, nop,TS val 100 ecr 700, dss dack8=1001 dsn8=101 ssn=0 dll=1 nocs fin, nop, nop>
++0.0   < . 1001:1001(0) ack 101 win 450 <dss dack8=102 nocs>
```

Comment:
should it not be better to also include the DATA_FIN here with this DACK validating the previous DATA_FIN?

Reply:
good point: let's see if  the server under test validates the incoming data fin also. I'll push an update to this PR